jsapp/scss/components/_kobo.form-builder.scss

```diff
@@ -218,7 +214,9 @@ $t-form-builder-aside-open: 200ms;
 
       button {
         color: $cool-gray;
-        overflow: visible; // for tooltip
+        overflow: visible;
+
+        // for tooltip
```

Comment:
something weird is happening with comments and this is second time I see it in your commit - is this some autofix? `// for tooltip` refers to `overflow: visible;`, so if not on same line, it should be above it :)

Reply:
Oh, yeah, I use a sass beautifier that messes these up.